po/de_DE.po

```diff
@@ -77,43 +77,103 @@ msgstr ""
 
 #: data/nl.hjdskes.gcolor3.appdata.xml.in:53
 msgid "Centrally manage colors between several running instances;"
-msgstr ""
+msgstr "Zentrale Farbverwaltung zwischen verschiedenen laufenden Instanzen;"
 
 #: data/nl.hjdskes.gcolor3.appdata.xml.in:54
 msgid ""
 "Follow the XDG directory specification for the configuration file. This "
 "means that the configuration file has moved from `~/.rgb.ini` to `"
 "$XDG_CONFIG_HOME/gcolor3/config.ini`;"
 msgstr ""
+"Die Konfigurationsdatei folgt nun der \"XDG directory specification\". Das "
+"bedeutet dass die Datei nun am Ort `$XDG_CONFIG_HOME/gcolor3/config.ini` "
+"statt am Ort `~/.rgb.ini` liegt;"
 
 #: data/nl.hjdskes.gcolor3.appdata.xml.in:56
 msgid "Copy the selected color's hex value from the tree view using Ctrl-C;"
-msgstr ""
+msgstr "Strg+C zum Kopieren des gewählten HEX-Werts aus der Baumansicht;"
 
 #: data/nl.hjdskes.gcolor3.appdata.xml.in:57
 msgid "Make the main window resizable;"
-msgstr ""
+msgstr "Hauptfenster größenveränderbar;"
 
 #: data/nl.hjdskes.gcolor3.appdata.xml.in:58
 msgid "Translation updates."
-msgstr ""
+msgstr "Aktualisierung der Übersetzungen."
+
+#: data/color-row.ui:46
+msgid "Click to change the color's name"
+msgstr "Klicken um den Namen der Farbe zu ändern"
+
+#: data/color-row.ui:47
+msgid "Name"
+msgstr "Name"
+
+#: data/color-row.ui:68
+msgid "Copy this color's hex value to the clipboard"
+msgstr "Diesen HEX-Wert in die Zwischenablage kopieren"
+
+#: data/color-row.ui:90
+msgid "Delete this color"
+msgstr "Diese Farbe löschen"
 
 #: data/menus.ui:7
+msgid "_Keyboard Shortcuts"
+msgstr "_Tastenkürzel"
+
+#: data/menus.ui:11
 msgid "_About"
 msgstr "_Über Gcolor3"
 
-#: data/menus.ui:11
+#: data/menus.ui:15
 msgid "_Quit"
 msgstr "_Beenden"
 
-#: data/window.ui:55
-msgid "Saved colors"
-msgstr "Gespeicherte Farben"
+#: data/shortcuts.ui:14
+msgid "General"
+msgstr "Allgemein"
+
+#: data/shortcuts.ui:19
+#, fuzzy
```

Comment:
You should remove any fuzzy comments when you have checked the translation.

Reply:
Aye, it's basically reminders for me but I can remove them anyway since I'll push a tweaked translation for 2.3.1 most probably anyway.

Reply:
It's OK to keep them as a reminder if you haven't checked the translation yet, but you should remove them once you have.